app/assets/javascripts/actions/index.coffee

```diff
@@ -0,0 +1,37 @@
+Screensmart.Actions =
+  addMessage: (message) ->
+    type: 'ADD_MESSAGE'
+    message: message
+
+  setAnswer: (key, value) ->
+    (dispatch) =>
+      dispatch @_setAnswer(key, value)
+      dispatch @updateResponse()
+
+  _setAnswer: (key, value) ->
+    type: 'SET_ANSWER'
+    key: key
+    value: value
+
+  updateResponse: ->
+    (dispatch, getState) ->
+      response = getState().response
+      syncResponse(response).then (data) ->
+        dispatch Screensmart.Actions.receiveResponseUpdate(data)
+
+  setResponseLoading: ->
+    type: 'SET_RESPONSE_LOADING'
+
+  receiveResponseUpdate: (data) ->
+    type: 'RECEIVE_RESPONSE_UPDATE'
+    response: data.response
+
+syncResponse = (response) ->
```

Comment:
Als deze methode punt 4 ook deed was het een prima naam geweest. 

Maar een post op /responses verwacht ik niets terug dan de net gecreerde response, mogelijk met wat defaults ingevuld, maar absoluut niet met een volgende vraag erin. 

Mogelijk is patch een betere verb om aan te geven dat het een apart geval is, hoewel je meestal post gebruikt voor non-rest gevallen, maar meestal heb je dan een pad met een action erin die duidelijk aangeeft wat er gaat gebeuren.